cmd/generate-lint/generate-lint.go

```diff
@@ -135,11 +135,15 @@ func generateTasks() (*shrub.Configuration, error) {
 			return nil, err
 		}
 	} else {
-		maxHosts = patchMaxHosts
 		targets, err = targetsFromChangedFiles(changes)
 		if err != nil {
 			return nil, err
 		}
+		if len(targets) == 1 {
+			maxHosts = 1
+		} else {
+			maxHosts = patchMaxHosts
```

Comment:
should we just set this to len(targets) since we don't really care if they run on different hosts?

Reply:
In that case, we should just set it to `len(targets)` for both patches and waterfall versions.

I removed the max host limit.